python/tank/util/yaml_cache.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) 2015 Shotgun Software Inc.
```

Comment:
Sorry, I wasn't paying proper attention. I was under the impression that on the yaml module there was a yaml.load method. I was suggesting that that method could have been decorated/monkey patched so that it's callers didn't have to be aware that there was a caching mechanism in place. However I just realized that this API is all stream based, so therefore the YamlCache object makes more sense since it's not the yaml.parse method that needs to be cached but the open part.

Sorry for the confusion.